src/frontend/helpers/extract-main-domain.ts

```diff
@@ -0,0 +1,15 @@
+export function extractMainDomain(url: string) {
+  try {
+    const domain = new URL(url).hostname
+    const parts = domain.split('.')
+
+    if (parts.length >= 2) {
+      return parts.slice(-2).join('.')
+    }
+
+    return domain
+  } catch (e) {
+    console.error(e)
```

Comment:
could we change this to a console warn and also log the url that it was called with as well. 

this is what I get on first app launch 
![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/27568879/ead6dbbd-cbea-4e48-b424-c25e8ef207e3)

Reply:
yes. I added an early return too in the `getStyles` function to avoid spamming the console with warnings